batavia/VirtualMachine.js

```diff
@@ -9,7 +9,21 @@ batavia.VirtualMachine = function(loader) {
 
     if (loader === undefined) {
         this.loader = function(name) {
-            return document.getElementById('batavia-' + name).text.replace(/(\r\n|\n|\r)/gm, "").trim();
+            var element = document.getElementById('batavia-' + name);
```

Comment:
This code looks good from a code clarity perspective, not quite sure what the code is doing.